usaspending_api/etl/management/commands/load_table_from_delta.py

```diff
@@ -164,44 +176,184 @@ def handle(self, *args, **options):
+                qualified_temp_table,
+                postgres_schema_def,
+                self.table_spec["column_names"],
+                postgres_seq_last_value,
+            )
 
-        summary_msg = f"Copying delta table {delta_table} to a Postgres temp table {temp_table}."
-        if postgres_table:
-            summary_msg = f"{summary_msg} The temp table will be based on the postgres table {postgres_table}"
-        self.logger.info(summary_msg)
+            self.logger.info(
+                f"Note: this has merely loaded the data from Delta. For various reasons, we've separated the"
+                f" metadata portion of the table download to a separate script. If not already done so,"
+                f" please run the following additional command to complete the process: "
+                f" 'copy_table_metadata --source-table {self.qualified_postgres_table} --dest-table {qualified_temp_table}'."
+            )
+
+        if delta_table_load_version_key:
+            update_last_staging_load_version(delta_table_load_version_key, last_update_version)
+
+        # We're done with spark at this point
+        if spark_created_by_command:
+            spark.stop()
+
+    def _surpassed_update_threshold(self, spark: SparkSession, df: DataFrame) -> bool:
+        """
+        Determines whether the number of records in the incoming DF exceeds the threshold of allowable
+        changes when compared to total number records in the table being copied. The threshold percentage
+        is based on the `incremental_parameter` parameter for the job.
+
+        Args:
+            spark: the active SparkSession to work within
+            distinct_df: Dataframe containing changed records from table
+        Returns:
+            threshold_surpassed: Whether or not the number of changes has exceeded the threshold
+        """
+
+        threshold = self.options["incremental_threshold"]
+        threshold_surpassed = False
+
+        total_record_count = spark.sql(f"SELECT COUNT(*) FROM {self.qualified_source_delta_table}").first()[0]
+
+        records_changed = df.count()
+
+        if records_changed / total_record_count > threshold:
+            self.logger.warning(f"Exceeded threshold of {threshold:.0%} for incremental updates")
+            self.logger.warning(f"---- Total Records  in  {self.qualified_source_delta_table}: {total_record_count}")
+            self.logger.warning(f"---- Records Changed in {self.qualified_source_delta_table}: {total_record_count}")
+            threshold_surpassed = True
+
+        return threshold_surpassed
+
+    def _prepare_temp_table(self, temp_schema: str, temp_table_suffix: str, schema_override=None):
+        """
+        Creates a temporary table in the Postgres database. This table will either be based on the
+        the Delta Table being copied from's corresponding Postgres table (`self.qualified_postgres_table`)
+        or a provided `schema_override` schema.
+
+        NOTES:
```

Comment:
I believe this "NOTES" section duplicates knowledge. What you describe here are behaviors of this method. Anytime the method changes we are going to have to remember to update this "NOTES" section.

Reply:
I think this information is important for the caller to know. (ex. what happens if the temp table being prepared already exists?)